backend/Messenger/Modules/Messenger.Subscriptions/Features/PurchaseSubscriptionCommand/PurchaseSubscriptionCommand.cs

```diff
@@ -0,0 +1,5 @@
+﻿using Messenger.Core.Requests.Abstractions;
+
+namespace Messenger.Subscriptions.Features.PurchaseSubscriptionCommand;
+
+public record PurchaseSubscriptionCommand(Guid UserId, string CardNumber, string ExpirationDate, string HolderName, string Cvv, uint Plan, uint Term) : ICommand<bool>;
```

Comment:
Если больше трех то красивее так
```suggestion
public record PurchaseSubscriptionCommand(
    Guid UserId,
    string CardNumber,
    string ExpirationDate,
    string HolderName,
    string Cvv,
    uint Plan,
    uint Term) : ICommand<bool>;
```